packages/message-relayer/package.json

```diff
@@ -7,6 +7,9 @@
   "files": [
     "dist/*"
   ],
+  "bin": {
+    "withdraw": "./src/exec/withdraw.ts"
```

Comment:
Used `./src/exec` instead of `./exec` because `tsconfig` was fighting with me and I didn't feel like figuring it out.

Reply:
I think for this to be included in the npm package, we will need to include it in the `files` list